pages/stable-version.html

```diff
@@ -173,25 +173,41 @@ <h5>Quickstart Installation Instructions</h5>
                 <p>Please note Uyuni requires FQDNs for the Server, Proxy and clients, so usage of a properly-configured DNS server is mandatory for client updates to work.</p>
                 <p>If you want to install Uyuni in lab mode, for testing, you may be interested in how to easily make use of libvirt's included dnsmasq DNS and DHCP server: <a href="https://github.com/uyuni-project/uyuni/wiki/Libvirt-DNS-and-DHCP-without-Avahi">Libvirt DNS and DCHP without Avahi</a>.</p>
 
-                <p>1. Update openSUSE Leap 15.5:</p>
-                <code>$ sudo zypper update</code>
+                <p>1. Update openSUSE Leap Micro 5.5:</p>
+                <code>$ transactional update</code>
 
-                <p>2. Configure a resolvable fully quallified domain name with <strong>Yast2 > System > Netwok Settings > Hostname/DNS</strong></p>
+                <p>2. Reboot the system</p>
 
-                <p>3. Add the repository for the server:</p>
-                <code>$ sudo zypper ar https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Server-POOL-x86_64-Media1/ uyuni-server-stable</code>
+                <p>3. Configure a resolvable fully quallified domain name (FQDN)</p>
 
-                <p>4. Refresh metadata from the repositories:</p>
-                <code>$ sudo zypper ref</code>
+                <p>4. Enter the transactional shell:</p>
+                <code>$ transactional-update shell</code>
 
-                <p>5. Install the pattern for the Uyuni server:</p>
-                <code>$ sudo zypper in patterns-uyuni_server</code>
+                <p>5. Add the container utility repository:</p>
```

Comment:
And neither is this one:
```suggestion
                <p>5. Add the Uyuni Server repository:</p>
```